genai-perf/genai_perf/config/input/config_command.py

```diff
@@ -149,9 +161,33 @@ def _check_output_format_and_generate_plots(self) -> None:
         ]:
             if self.output.generate_plots:
                 raise ValueError(
-                    "User Config: generate_plots is not supported with the {self.endpoint.output_format} output format"
+                    f"User Config: generate_plots is not supported with the {self.endpoint.output_format} output format"
```

Comment:
Thanks for this cleanup! Are you adding unit testing for error handling? We have test_conditional_errors in test_cli.py, which should hopefully be usable for the config now that the checking is being consolidated. We should be unit testing all the errors we throw somewhere centralized.

Reply:
Yes, All unit testing for error handling is currently done in test_cli. This is how the missing f-string was uncovered.